### ISSUE TITLE
Hello from Typeform! 

### ISSUE THREAD

**Issue by shubhaswamy** [isabellachen/typeform-messenger-server#1]
Hi Isa! :wave:

My name is Shubha and I'm a developer advocate at Typeform. We recently came across your project, and wanted to reach out because we found the project interesting. It's awesome that you've built this cool messenger bot with Typeform's API! 

What was your experience with using Typeform's API? Did you encounter any issues, or do you have feedback? We would love to hear from you :grinning:

Would you be interested in writing for our [engineering blog](https://medium.com/typeforms-engineering-blog) about your experience, or a blog post about building this messenger bot? We love having community members share their experience about using Typeform! 

If you need to reach out to me, please feel free to contact me at shubha.swamy@typeform.com. :grinning:

**Comment by isabellachen** [comment]
Hi Shubha!

Thank you for your emails.

In the end, I downscaled the project so instead of an entire server which
outputs typeform form to facebook messenger format, saving the responses
from facebook, I focused on the part which translates typeform form fields
to facebook responses, leaving how the responses from the user on Facebook
are saved up to the developer.

Let me think about the questions you've asked and give you a detailed
answer once I've had time to think about them. I would also love to write
about my experience working with Typeform for your engineering blog.

Funnily enough, I work at Codeworks which is beside one of the Typeform
offices in Barcelona, the one on Carrer d'Avilla, do you happen to work
there by any chance?

Cheers,
Isabella

**Comment by shubhaswamy** [comment]
That's awesome! Unfortunately I do not work at Typeform's office in Barcelona, I actually work in the San Francisco office. 

That would be awesome if you could get back to me about your thoughts and writing for our blog. I'd love to continue this conversation over email, please feel free to email me at shubha.swamy@typeform.com.

**Comment by isabellachen** [comment]
Hi! So I've attached a blog post to this email, let me know your thoughts.

As for feedback about TF, I didn't have an problems working with the API.
The only issue I had was when creating a new application and giving the
authorization callback uri https://admin.typeform.com/account#/section/apps ...
the suggestion was to use https://mycompany.com/authorize  which didn't
work out for me. This was an issue and I posted my solution to it
https://github.com/Typeform/results-example/issues/7 . But that was the
only problem I encountered.
